### ISSUE TITLE
Review WARN logs during tile build

### ISSUE THREAD

**Issue by nilsnolde** [valhalla/valhalla#3984]
There's a few WARN logs which _really_ clutter the build logs _immensely_. During parsing it's mostly the `layer` tag, e.g.: `2023/02/21 20:59:07.76195^@ [WARN] Error during parsing of `layer` tag on the way 1147187673
: stoi`. 

Currently, there's been some sort of vandalism/mass import which duplicated 100s of 1000s invalid tags: https://www.openstreetmap.org/changeset/132695116. 

But even in the best of times this is making the logs less useful. There's other contenders like time conditionals I think or during enhancement the "local edges exceeded 8" or whatever (and those are less useful IMHO).

I'm not even sure if we can reasonably do smth about it since most are of course valid warnings. Would it be feasible/useful to extend the logging to be able to use stdout until WARN and stderr from WARN on?

**Comment by kevinkreiser** [comment]
why not demote to log debug. Release builds have that disabled but it's there if you want it as a cmake argument

**Comment by nilsnolde** [comment]
Yeah, debug would be fine for that one too. It's not even used much as far as I can tell.. The code seems to look for a `preferred_layer` in the locations (undocumented..) and filter it according to the way's `layer` tag. 

I'll go through the biggest log offenders and see if it makes sense to demote to debug. I'd be really nice to be able to navigate the logs more efficiently.
